packages/typography/src/styled/StyledCodeBlock.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { ThemeProps, DefaultTheme, css } from 'styled-components';
+import { DEFAULT_THEME, getColor, retrieveComponentStyles } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'typography.codeblock';
+
+const colorStyles = (props: IStyledCodeBlockProps & ThemeProps<DefaultTheme>) => {
+  const backgroundColor = getColor('neutralHue', props.isLight ? 100 : 1000, props.theme);
```

Comment:
@steue note that the use of `1000` here awaits the decision on `grey-900`.

Reply:
@jzempel design team's consensus is to use `getColor`, as `grey-900` one off isn't ideal and should be visited holistically.

Reply:
Awesome. I'm a bit relieved.